server/middleware/tchannel-client/helpers/make-channels.js

```diff
@@ -22,10 +22,14 @@
 const path = require('path');
 const isIPv4 = require('is-ipv4-node');
 const TChannelAsThrift = require('tchannel/as/thrift');
-const { PEERS } = require('../constants');
+const { PEERS_DEFAULT } = require('../constants');
 const lookupAsync = require('./lookup-async');
 
 const makeChannels = async client => {
+  const PEERS = process.env.CADENCE_TCHANNEL_PEERS
+    ? process.env.CADENCE_TCHANNEL_PEERS.split(',')
+    : PEERS_DEFAULT;
+
```

Comment:
broadly makes me think that exposing a getter rather than default values would be better... but far from necessary when there's just one use.

Reply:
I will refactor further in a future PR. I was thinking of rising all application variables to the init and can optionally pass a configuration or set these globals. Then that way it all flows the same way and preserves backwards compatibility.